samples/java/com/microsoft/bing/commerce/samples/CutomizationAPISample.java

```diff
@@ -0,0 +1,496 @@
+package com.microsoft.bing.commerce.samples;
```

Comment:
Same as above, let's move this to the [central repo](https://github.com/microsoft/bing-commerce)